packages/block-editor/src/hooks/index.js

```diff
@@ -29,11 +29,13 @@ import childLayout from './layout-child';
 import contentLockUI from './content-lock-ui';
 import './metadata';
 import blockHooks from './block-hooks';
+import blockBindingsPanel from './block-bindings-panel';
```

Comment:
Maybe we can call this `blockBindgins` to keep consistency with `blockHooks`

Reply:
Done!